### ISSUE TITLE
Mandatory filelink.HARDLINK

### ISSUE THREAD

**Issue by mottosso** [tokejepsen/filelink#2]
Spontaneously, I'd expect to just need two arguments.

```python
filelink.create("c:/path/to/file.py", "c:/path/to/link.py")
```

Having `HARDLINK` as a third argument does make it explicit, but I wonder whether it's too explicit. Should we make hard links a default? One on hand, it would align with [`os.link`](https://docs.python.org/2/library/os.html#os.link) but on the other I think "soft" or "symbolic" links are more common in both Windows and Linux-land and might be what is expected.

**Comment by tokejepsen** [comment]
Good point.

As you said I was under impression that "soft" links are more common, so it seems like we might "just" be catering for ourselves.
In the end though, I don't think it matters too much as long as there are some good documentation for it.

**Comment by tokejepsen** [comment]
Resolved with #8
